internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -80,6 +81,7 @@ If the argument holds an image tag, it must be present remotely.`,
 	scorecardCmd.Flags().StringVarP(&c.namespace, "namespace", "n", "", "namespace to run the test images in")
 	scorecardCmd.Flags().StringVarP(&c.outputFormat, "output", "o", "text",
 		"Output format for results. Valid values: text, json, xunit")
+	scorecardCmd.Flags().StringVar(&c.podSecurity, "pod-security", "restricted", "option to run scorecard with legacy pod security context")
```

Comment:
For now I think we are reverting to `legacy` by default, so I think we should change this to be:
```suggestion
	scorecardCmd.Flags().StringVar(&c.podSecurity, "pod-security", "legacy", "option to run scorecard with legacy pod security context")
```

for more context see: https://github.com/operator-framework/operator-sdk/pull/6210#issue-1473321069

Reply:
Updated.